src/lib/equinox.ts

```diff
@@ -21,9 +21,6 @@ export function launch(address: AddressInfo, application: Application) {
         case 'darwin':
             platform = 'macos';
             break;
-        case 'win32':
-            platform = 'windows';
-            break;
```

Comment:
could you please explain this change?

Reply:
each platform require building a separate server bundle, currently we build server bundle only for macOS platform (Tycho plugin), so if we run the extension on windows there will not be appropriate backend to handle requests